api/localDevAuth.ts

```diff
@@ -49,7 +50,26 @@ export async function fetchScopeData(
   scopeGroup: string
 ): Promise<ScopeData> {
   return http.get<ScopeData>(accountId, {
-    url: `localdevauth/v1/auth/check-scopes`,
+    url: `${LOCALDEVAUTH_API_AUTH_PATH}/check-scopes`,
     params: { scopeGroup },
   });
 }
+
+export async function fetchAppInstallationData(
+  portalId: number,
```

Comment:
Nit, we've been using `accountId` in this repo. Can you switch the arg to use that naming?